sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/spark_helpers.py

```diff
@@ -2,9 +2,11 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 # pylint: disable=protected-access
-
+import re
```

Comment:
we should have a changelog line for this PR.

Reply:
added